pyspeckit/cubes/SpectralCube.py

```diff
@@ -673,6 +684,11 @@ def fit_a_pixel(iixy):
             else:
                 max_sn = None
             sp.specfit.Registry = self.Registry # copy over fitter registry
+            # Do some homework for local fits
+            xpatch = [1,1,1,0,0,0,-1,-1,-1]
+            ypatch = [1,0,-1,1,0,-1,1,0,-1]
```

Comment:
should these be arrays for use below?  `ypatch+y` could result in `TypeError: can only concatenate list (not "int") to list` or similar otherwise